chimney/src/main/scala/io/scalaland/chimney/internal/macros/TransformerMacros.scala

```diff
@@ -619,12 +650,13 @@ trait TransformerMacros extends TransformerConfigSupport with MappingMacros with
       config: TransformerConfig
   )(From: Type, To: Type): Either[Seq[DerivationError], Tree] = {
 
+    // TODO reimplement using wrapper classes and lifted transformers
```

Comment:
let's talk Monday on what your idea is :) 
Meanwhile, not sure we want to have todo comments on master. Ticket would be better: we'll be able to keep track of and prioritise the issue

Reply:
Basically, lifted transformers in Chimney allow transforming values wrapped in arbitrary wrappers, given there's a typeclass implementation for them.

The idea is that perhaps instead of modifying the macro to extract values from the proto wrappers, we could define a typeclass and use `TransformerF` instead. Just an idea worth checking, nothing definite yet (haven't had time to play with it)